### PR TITLE
Pia 1411

### PR DESCRIPTION
**Pull request by zladzeyka**
* Updated GiniPayBank interface ( added lost completion, remove var for redirect scheme)
  * Added example app ->Bank target
  * To run the example:
   - pod install/update;
   - add credentials in plist

### CODE REVIEW

File: Example/Bank/PaymentViewController.swift [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+            }
+        }
+        
+        viewModel?.onPaymentInfoFetched = {[weak self] paymentInfo in
+            DispatchQueue.main.async {
+                self?.fillOutTheData(paymentInfo: paymentInfo)
+            }
+        }
+        
+        viewModel?.onResolvePaymentRequest = {
```
Comment (a-szotyori) [inline Example/Bank/PaymentViewController.swift:92]: Should this also use `[weak self]`?

File: Example/Bank/PaymentViewController.swift [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+            DispatchQueue.main.async {
+                self.backToBusinessButton.isHidden = false
+                self.showAlert(message: "Payment was successfully resolved")
+            }
+        }
+        viewModel?.onResolvePaymentRequestErrorHandling = {
```
Comment (a-szotyori) [inline Example/Bank/PaymentViewController.swift:98]: Should this also use `[weak self]`?

File: Example/Bank/PaymentViewController.swift (hunk 1 above)
Comment (zladzeyka) [reply Example/Bank/PaymentViewController.swift:92]: Thanks for noticing it

File: Example/Bank/PaymentViewController.swift (hunk 2 above)
Comment (zladzeyka) [reply Example/Bank/PaymentViewController.swift:98]: yes

**Review by a-szotyori — APPROVED** [review]

**Pull request merged**